pkgs/by-name/sh/shadps4/package.nix

```
@@ -0,0 +1,139 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  alsa-lib,
+  boost184,
+  cmake,
+  cryptopp,
+  glslang,
+  ffmpeg,
+  fmt,
+  jack2,
+  libdecor,
+  libpulseaudio,
+  libunwind,
+  libusb1,
+  magic-enum,
+  mesa,
+  pkg-config,
+  pugixml,
+  qt6,
+  rapidjson,
+  renderdoc,
+  sndio,
+  toml11,
+  vulkan-headers,
+  vulkan-loader,
+  vulkan-memory-allocator,
+  xorg,
+  xxHash,
+  zlib-ng,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation {
+  pname = "shadps4";
+  version = "0.3.1-unstable-2024-10-13";
```

With fixed update script, it disagrees on the version:

```
+ update-source-version shadps4 0.3.0-unstable-2024-10-13 --rev=bd9f82df94847b4a5f3d2676ae938f064505c992
update-source-version: error: Failed to replace source revision 'bd9f82df94847b4a5f3d2676ae938f064505c992' to 'bd9f82df94847b4a5f3d2676ae938f064505c992' in 'shadps4'!
```

Base version should be the latest released version, so [0.3.0](https://github.com/shadps4-emu/shadPS4/releases/tag/v.0.3.0).

```suggestion
  version = "0.3.0-unstable-2024-10-13";
```